### ISSUE TITLE
There is no rms information for the test set in the lcurve.out

### ISSUE THREAD

**Issue by jinfeng-data** [deepmodeling/deepmd-kit#781]
**Summary**
There is only the rms information for the training set, but no rms information for the test set in the lcurve.out.

We are using the deepmd-kit-2.0.0.b1-cuda11.1_gpu version to train a potential energy surface. The dp train can run normally, and the training process also seems normal. However, in the lcurve.out during the training procedure, there is no output of the rms information for the test set, but only the rms for training set as the following,
" #  step      rmse_trn    rmse_e_trn    rmse_f_trn         lr
      0      2.77e+01      6.40e-01      8.74e-01    1.0e-03
    100      1.39e+01      2.22e-01      4.39e-01    1.0e-03
    200      8.11e+00      1.61e-01      2.56e-01    1.0e-03
    300      6.70e+00      2.64e-02      2.12e-01    1.0e-03
    400      8.27e+00      6.61e-03      2.61e-01    1.0e-03
    500      5.81e+00      1.18e-02      1.84e-01    1.0e-03"

Normallly in the previous versions, the output in the lcurve.out is like the following,
"# batch      l2_tst    l2_trn    l2_e_tst  l2_e_trn    l2_f_tst  l2_f_trn         lr
      0    3.84e+01  3.95e+01    1.03e+01  1.03e+01    8.22e-01  8.74e-01    1.0e-03
    100    2.68e+01  2.66e+01    6.05e-01  5.98e-01    8.46e-01  8.38e-01    1.0e-03
    200    2.51e+01  2.58e+01    8.94e-02  8.85e-02    7.95e-01  8.16e-01    1.0e-03
    300    2.16e+01  2.05e+01    7.43e-02  7.17e-02    6.82e-01  6.49e-01    1.0e-03
    400    1.73e+01  1.76e+01    2.43e-02  3.02e-02    5.48e-01  5.56e-01    1.0e-03
    500    1.49e+01  1.44e+01    1.14e-01  1.08e-01    4.70e-01  4.55e-01    1.0e-03"

So, I am wondering whether there is a bug in deepmd-kit-2.0.0.b1-cuda11.1_gpu-Linux-x86_64, or my input is not correct. My input file has been attached. Please help to check it. Thanks very much !

**Deepmd-kit version, installation way, input file, running commands, error log, etc.**
deepmd-kit-2.0.0.b1-cuda11.1_gpu-Linux-x86_64

cuda11.1

**Steps to Reproduce**

**Further Information, Files, and Links**

[input.zip](https://github.com/deepmodeling/deepmd-kit/files/6685643/input.zip)

**Comment by njzjz** [comment]
This is a breaking change in v2.0. Now all dataset is considered as the training set, and you need to manually set validation set.
https://github.com/deepmodeling/deepmd-kit/blob/44d49f346dbab84723db0b086e9bac9c5792abd7/examples/water/se_e2_a/input.json#L45-L55

**Comment by njzjz** [comment]
See also https://deepmd.readthedocs.io/en/latest/train-se-e2-a.html#training-parameters.
